prime-react/src/main/scala/react/primereact/Button.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.primereact
+
+import cats.syntax.all.*
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import react.common.*
```

Comment:
i've been trying to limit the use of `react.common.*` because the existing syntax is imported automatically and I'v seen a few conflicts using them
I moved some to this package:
https://github.com/gemini-hlsw/lucuma-ui/tree/master/modules/ui/src/main/scala/lucuma/ui/syntax
Maybe we should move them here and try to use them instead of the "legacy" syntaxis?

Reply:
Can you elaborate?

What conflicts did that import cause?

Is your suggestion to use `import lucuma.ui.syntax.render.*` instead?

Reply:
When I tried to port explore to scala3 there would be conflicts using the syntax in `react.common.*`.
Thus I did a more limited set of syntax which are now in `lucuma.ui.syntax.render`

They could be moved here but once I tried I got hundreds of errors
I think we may want to move them here into a different package and at least for the new facades use it instead of `react.common.*`

Reply:
Ok, we can take a look at it. I find it quite convenient to just `import react.common.*` and have all machinery kick-in, but of course it stops being convenient if there are conflicts.

Reply:
We could also do it in one go in the near future once we get rid of SUI. which is the source of 90% of the errors
It maybe simpler that way?

Reply:
We can't really use `lucuma.ui.syntax.render` here, because `lucuma-ui` depends on `lucuma-react`.